src/frogger.cpp

```diff
@@ -54,13 +54,19 @@ leapfrog::TensorMap1<int> get_age_groups_hiv_span(const Rcpp::List projection_pa
   return age_groups_hiv_span;
 }
 
+std::vector<int> parse_output_steps(Rcpp::NumericVector output_steps, int proj_years) {
```

Comment:
what is being parsed here? why is `proj_years` an input that is not used? should you also verify that these are (strictly) increasing?

Reply:
We don't need proj_years - good catch. I think that was just some old code left around as I originally had this interface take a time step and was then building `output_steps` from the time step and `proj_years`. By parsing here I just really mean converting from R type to C++ type. I've tried updating that name, this is now `transform_output_steps` and I've updated the other functions similar to this one which are also converting from R type to C++ type, possibly doing some validation on the way.

I can check they are increasing but I think this will work atm even if they aren't. If someone passed in `seq(60, 0, -1)` it would return with time backwards if someone really wanted. I can explicitly disallow if you think it makes interface tidier